s32/mcux/devices/S32K344/S32K344_features.h

```diff
@@ -21,6 +21,8 @@
 #define FSL_FEATURE_SOC_LPSPI_COUNT (6)
 /* @brief EDMA availability on the SoC. */
 #define FSL_FEATURE_SOC_EDMA_COUNT (1)
+/* @brief DMAMUX availability on the SoC. */
+#define FSL_FEATURE_SOC_DMAMUX_COUNT (1)
```

Comment:
The chip has two DMAMUX instances:

![image](https://github.com/user-attachments/assets/2ab37735-3631-49f7-988d-6dde32a3bdf4)